flink-tests/src/test/java/org/apache/flink/api/datastream/DataStreamCollectTestITCase.java

```diff
@@ -61,6 +64,11 @@ public void testStreamingCollect() throws Exception {
 	@Test
 	public void testStreamingCollectAndLimit() throws Exception {
 		StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+
```

Comment:
Maybe add that in a new test case because now we don't test the `STREAMING` use case anymore?

Reply:
Fo Sho